### PR TITLE
Create `LoginView`

### LINKED ISSUE #4: Create Sign In screen view
## Depends on tasks:
- #80
- #31 

## Description
Create a view that allows a user to select how to run the app  - as a pet owner or a pet sitter.
It's a temporary view to manage 2 flows in the app (pet owner and pet sitter).

### User story:

- Create View to display two buttons for (log in as pet owner and pet sitter);
- The `I am an owner` button redirects to the main tab screen for a pet owner;
- The `I am a sitter` button redirects to the main tab screen for a pet sitter;
- Pressing on the button triggers ViewModel to save the selected role as user Preferences.

## UI Mocks
Create a screen with 2 buttons with text `I am an owner` and `I am a sitter`.
You can choose your own design based on [App style guidelines](https://github.com/ios-course/ironfoudation-team-project/wiki/App-style-guidelines).

### PR DESCRIPTION
**Pull request by marinkman**
## Summary
This PR is a part of the work to resolve #4 

## UI Preview (Screenshots)
<details>
  <summary> Tap to show screenshot </summary>

Screenshot 2023-06-15 at 15 09 51

</details>

## Changes
- Created `PurpleCapsuleOfInfinityWidth` as `ButtonStyle`
- Created `LoginView`
- Applied `LoginView` in the `WoofApp` to the `.none` case

## Checks to complete
- [x] I've built and run my changes, and no warnings or errors occurred.
- [x] All existing tests passed with my changes.
- [x] My code follows our style guide.
- [x] I've performed a self-review of my code.
- [x] PR's title has a conventional commit style (short and descriptive).
- [x] Every non-private interface is documented properly.
- [x] PR has assignee, reviewers, milestone, it is properly linked to the project and to the issue.

### CODE REVIEW

File: Woof/Woof/View/LoginView.swift [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import SwiftUI
+
+/// A view that allows the user to select a role in the application.
+/// Based on the selected role, the user will be directed to the appropriate app flow.
```
Comment (kirpichonok) [inline Woof/Woof/View/LoginView.swift:4]: Are you sure that view knows something about the app navigation?

Comment (marinkman) [reply Woof/Woof/View/LoginView.swift]: I removed this line.

File: Woof/Woof/View/LoginView.swift [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+                ) {
+                    Button(sitterButtonText) {
+                        viewModel.set(.sitter)
+                        directToTheSitterFlow.toggle()
+                    }
+                    .buttonStyle(
+                        PurpleCapsuleOfInfinityWidth()
+                    )
+                }
+            }
```
[the remark this answers is not in the document]
Comment (marinkman) [reply Woof/Woof/View/LoginView.swift]: Yes, thanks.
Done.

**Review by julyakmita — COMMENTED** [review]
Nice job!) One question:

File: Woof/Woof/View/LoginView.swift [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    }
+
+    // MARK: - Private interface
+
+    private var viewModel = LoginViewModel()
+
+    @State private var directToTheOwnerFlow = false
+    @State private var directToTheSitterFlow = false
+
+    private let ownerButtonText = "I am an owner"
```
[the remark this answers is not in the document]
Comment (marinkman) [reply Woof/Woof/View/LoginView.swift:43]: Agree, good idea.

[the remark this answers is not in the document]
Comment (marinkman) [reply Woof/Woof/View/LoginView.swift]: Done.

**Pull request merged**